### ISSUE TITLE
Tools sometimes activate when holding down the middle mouse button

### ISSUE THREAD

**Issue by Sleuth56** [toger5/TheBoard#4]
When moving around a whiteboard tools especially the eraser keeps deleting anything under the mouse cursor.

**Comment by toger5** [comment]
Thanks a lot. I was noticing some issues. just couldnt phrase what it was caused by.

**Comment by Sleuth56** [comment]
This happens on every tool to date except for the pen tool. This includes the new text tool.

**Comment by toger5** [comment]
This does not happen to me anymore. Is this issue still present for you?

**Comment by Sleuth56** [comment]
Yes, it does still happen for me. When panning using the middle mouse button the text, rectangle, eraser and line tools all do this.
I'm using Firefox version 91.0.2 (64-bit) and Manjaro linux.

**Comment by toger5** [comment]
Just to be sure: you use a normal mouse (not a trackpad or other input device).
Tbh i haven't tested with a normal mouse yet and also did not think about the fact that a normal mouse is handled by different logic. So this might be the root of the issue.

**Comment by Sleuth56** [comment]
I'm using a Logitech M570 ball mouse. But I have a normal USB mouse that shows the same issues.

https://user-images.githubusercontent.com/34520077/134248039-1ebac6cf-4c68-4caf-a70b-35648274df7f.mov
